tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature

```diff
@@ -88,25 +88,31 @@ Feature: move files
     Given user "Brian" has been created with default attributes
     And user "Brian" has shared folder "simple-folder" with user "Alice" with "read" permissions
     And user "Alice" has logged in using the webUI
-    When the user tries to move file "lorem.txt" into folder "simple-folder (2)" using the webUI
+    When the user opens the file action menu of file "lorem.txt" using the webUI
+    And the user selects move action for file "lorem.txt" using the webUI
+    And the user selects the folder "simple-folder (2)" as a place to move the file using the webUI
     Then it should not be possible to move into folder "simple-folder (2)" using the webUI
 
   Scenario: cancel moving a file
     Given user "Alice" has logged in using the webUI
     And the user has browsed to the files page
-    When the user tries to move file "data.zip" into folder "simple-empty-folder" using the webUI
+    When the user opens the file action menu of folder "data.zip" using the webUI
+    And the user selects move action for folder "data.zip" using the webUI
+    And the user selects the folder "simple-empty-folder" as a place to move the file using the webUI
     And the user cancels the attempt to move file into folder "simple-empty-folder" using the webUI
     Then file "data.zip" should be listed on the webUI
     But  file "data.zip" should not be listed in the folder "simple-empty-folder" on the webUI
 
   Scenario: cancel moving of multiple files at once
     Given user "Alice" has logged in using the webUI
     And the user has browsed to the files page
-    When the user tries to batch move these files into folder "simple-empty-folder" using the webUI
+    When the user marks these files for batch action using the webUI
       | file_name   |
       | data.zip    |
       | lorem.txt   |
       | testapp.zip |
+    And the user clicks the move button to move files using the webUI
```

Comment:
I would not prefer using click here, as we do not use it normally. May be we can use select here. You can decide about what would be better.